crates/polars-ops/src/series/ops/is_first.rs

```diff
@@ -97,6 +113,11 @@ pub fn is_first(s: &Series) -> PolarsResult<BooleanChunked> {
         },
         #[cfg(feature = "dtype-struct")]
         Struct(_) => return is_first_struct(&s),
+        #[cfg(feature = "group_by_list")]
+        List(inner) if inner.is_numeric() => {
```

Comment:
`if inner.is_numeric()` here is for more friendly error message. If there is a `List<Utf8>` here, we will get `InvalidOperationError: `is_first` operation not supported for dtype list[str]` instead of `ComputeError: "grouping on list type is only allowed if the inner type is numeric"`.